src/core/styles.components.css

```diff
@@ -53,6 +53,38 @@
     @apply -my-0.5 sm:-my-1 md:-my-2;
   }
 
+  .ui-section-spacing-collapse-order-first-full {
```

Comment:
@aralovelace I know I'm a bit late to the party with this review, however I think this falls short of my intentions. We should be making use of `:first-child` and `:last-child` pseudo selectors, because that means we don't need to be doing the whole `const collapseSpaceMap = {...}` business elsewhere.

@jamiehenson wdyt?

Reply:
I guess I don't know the whole picture of what this is for, but yes, using psuedoclasses seems preferable than introducing new classes, in theory